todo-app/src/TodoDesc.js

```diff
@@ -2,9 +2,31 @@ import React from 'react'
 import './todo.css'
 
 export default class TodoDesc extends React.Component {
+    constructor(props) {
+        super(props)
+    }
+    editTodo = (event) => {
+        this.props.controlActions.editToDo(this.props.index)
```

Comment:
You could use the todo id instead of index. We are safer using something more unique instead of the index.